frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1644,6 +1847,44 @@ mod tests {
 			assert!(MultiPhase::snapshot_metadata().is_none());
 			assert!(MultiPhase::desired_targets().is_none());
 			assert!(MultiPhase::queued_solution().is_none());
+			assert!(MultiPhase::signed_submissions().is_empty());
```

Comment:
What exactly is `is_empty()` called on? I don't see an impl for the method on `SignedSubmissions`

Reply:
https://github.com/paritytech/substrate/blob/bf4572c967a489b5a07d617aa8374887ec62f7a9/frame/election-provider-multi-phase/src/signed.rs#L310-L311

https://github.com/paritytech/substrate/blob/bf4572c967a489b5a07d617aa8374887ec62f7a9/frame/election-provider-multi-phase/src/signed.rs#L98-L99

`Deref` means that this type can use all the methods of its `Target`. In this case, its target is a `BoundedBTreeMap` which is the fundamental backing item of `SignedSubmissions`.